libmamba/src/core/download.cpp

```diff
@@ -1,26 +1,104 @@
 #include "mamba/core/download.hpp"
+#include "mamba/core/invoke.hpp"
+#include "mamba/core/thread_utils.hpp"
 #include "mamba/core/util.hpp"
+#include "mamba/core/util_scope.hpp"
+#include "mamba/util/build.hpp"
 #include "mamba/util/iterator.hpp"
 #include "mamba/util/string.hpp"
 #include "mamba/util/url.hpp"
+#include "mamba/util/url_manip.hpp"
 
 #include "curl.hpp"
 #include "download_impl.hpp"
 
 namespace mamba
 {
 
+    namespace
+    {
+
+        std::array<const char*, 6> cert_locations{
```

Comment:
`const` and/or `statiic const` or `static constexpr`?

Reply:
Should be constexpr, I've lost it in the rebase / squashing, good catch! Non need for static since it is in an anonymous namespace.